tests/autoscaler/pool_manager_test.py

```diff
@@ -503,6 +504,12 @@ def test_compute_new_resource_group_targets_scale_up_stale_pools_0(non_stale_cap
     assert new_targets == {'sfr-0': 2, 'sfr-1': 2, 'sfr-2': 2, 'sfr-3': 0, 'sfr-4': 0, 'sfr-5': 0, 'sfr-6': 0}
 
 
+def test_compute_target_capacity_no_resource_groups_found(mock_pool_manager):
+    mock_pool_manager.resource_groups = []
```

Comment:
Although it doesn't _really_ matter for this test (since all that matters is that `resource_groups` is "Falsy"), this should be a dict so that it has the same type as it would in real operation.

Reply:
Latest commit changes type to dict. Resolving.